package.json

```diff
@@ -24,7 +24,9 @@
     "test": "npm run jest",
     "jest": "cross-env MOCK_REDIS=1 jest",
     "start": "node src",
-    "debug-server": "nodemon server.js"
+    "debug-server": "nodemon server.js",
+    "webhint": "hint http://localhost:3000",
```

Comment:
Because `webhint` requires the server to be running, you should make sure it's up first.  We'll need to add [npm-run-all](https://www.npmjs.com/package/npm-run-all) to do this in a cross platform way (i.e., vs using `&&`):

```
npm install --save-dev npm-run-all
```

Then you can do:

```
"hint": "hint http://localhost:3000",
"webhint": "cross-env PORT=3000 npm-run-all -r -p start hint",
```

So this defines a `hint` and `webhint` script.  The `webhint` is what we'd use to run things, and it defines the server port to use as 3000, then runs two things in parallel: our server (`start`) and also `hint`

Reply:
https://webhint.io/docs/user-guide/development-flow-integration/local-server/ discuss this.